src/content/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui.mdx

```diff
@@ -72,11 +72,11 @@ import distributedtracingFocusonExample from 'images/distributed-tracing_screens
 
 [Distributed tracing](/docs/apm/distributed-tracing/getting-started/introduction-distributed-tracing) helps you monitor and analyze the behavior of your distributed system. After you enable distributed tracing, you can use our UI tools to search for traces and analyze them.
 
-For example, let's say you are an engineer troubleshooting errors in a complex transaction spanning many services. Here's what you can do in our UI:
+If, for example, you're looking to troubleshoot errors in a complex transaction spanning many services, here's what you can do in our UI:
```

Comment:
I think you can go ahead and cut out the `here's what you can do in our UI` and just say something like `you can review a span for a request to locate the alerting service:` (idk if this is what the steps actually do, just a suggestion for sentence construction